css/main.css

```diff
@@ -1,19 +1,158 @@
+}
+
 .hero-section {
-    background-image: url(../images/bg-hero.jpg);
-    background-size: cover;
-    min-height: 90vh;
+  background-image: url(../images/bg-hero.jpg);
+  background-size: cover;
+  min-height: 95vh;
 }
 .navbar-light .navbar-nav .nav-link {
-    color: white;
+  color: white;
 }
 .navbar-light .navbar-nav .nav-link:hover {
-    color: white;
-    text-decoration: underline;
+  color: white;
+  text-decoration: underline;
 }
 .main-form {
-    padding: 50px;
-    width: 400px;
-    background-color: rgba(255, 255, 255, 0.8);
-    border: solid 1px #d4d4d4;
+  padding: 35px;
+  width: 400px;
+  background-color: rgba(255, 255, 255, 0.8);
+  border: solid 1px #d4d4d4;
 
-}
\ No newline at end of file
+}
+#cta-button{
+border-radius: 3px;
+background-color: #cc5054;
+box-shadow: 0 3px 0 0 #940308;
+border: solid 1px #cc5054;
+}
+.info {
+  font-family: ProximaNova;
+  font-size: 18px;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.33;
+  letter-spacing: normal;
+  text-align: left;
+  color: #333333;
+}
+
+.bioPic {
+  width:200px;
+  height: 296px;
+}
+.h1{
+  margin-top:99px;
+}
+.skillsPic {
+  height: 250px;
+  width: 350px;
+}
+.accomPic {
+  height: 250px;
+  width: 350px;
+}
+ul li.accomplishment::before {
+  content: url("../images/check.svg");
+  height: 10px;
+  width: 10px;
+  margin-right:  0px;
+}   
+li.accomplishment {
+  margin-left: 0px;   
+}
+ul {
+  list-style-type: none;
+}
+p {
+padding-bottom : 5px;
+}
+.carousel-content {
+  color:black;
+  display:flex;
+
+}
+
+#text-carousel {
+width: 100%;
+position: relative;
+padding: 70px;
+}
+.testimonials {
+background-color: #f5f5f5;
+}
+
+.carousel-inner {
+  min-height: 150px;
+  width: 90%;
+}
+.carousel-control-prev-icon {
+  content: url(../images/arrow.svg);
+  width: 49px;
+  height: 82px;
+  background-image: none;
+}
+.carousel-control-next-icon {
+  content: url(../images/arrow.svg);
+  width: 49px;
+  height: 82px;
+  transform: rotate(180deg);
+  background-image: none;
+}
+.carousel-item {
+  max-width: 88%;
+}
+
+.carousel-slide {
+  max-height: 415px;
+}
+.carousel-item::before {
+  content: url(../images/quotes.svg);
+  width: 45px;
+  height: 36px;
+  position: relative;
+  z-index: 1;
+  padding-left: 17%;
+}
+.testims {
+  padding-left: 25%;
+  width: 100%;
+}
+.card-img-top {
+  height:300px;
+}
+.card-text {
+  display: none;
+}
+.card-img:hover + .card-text {
+  display: block;
+  z-index: 100;
+  position: absolute;
+  color: white;
+  font-size: 18px;
+  font-weight: bold;
+  font-family: ProximaNova;
+}
+footer {
+  height: 60px;
+  line-height: 46px;
+}   
+#indicator-pic1,
+#indicator-pic2 {
+  width: 90px;
+  height: 90px;
+}
+#testimonials .carousel-indicators li {
+  margin-right: 10%;
+  margin-top: -10%;
+  height: 90px;
+  width: 90px;
+  border-radius: 50%;
+}
+#testimonials .carousel-indicators img:active {
+  border: 2px solid #cc5054;
+  background-image: none;
+}
```

Comment:
at the end of the code there should be an empty string.

Reply:
@melnichenkod This is a very good point. VS code can do it automatically if you set it User Settings. All you need to do is add this one line:
Steps:
1. Press CMD(CTRL) + ','
2. Paste the following line
```
"files.insertFinalNewline": true
```
Here is my example: http://prntscr.com/iom5cd

Reply:
Ok, thank you!

Reply:
However, it doesn't seem to be recognized while making a commit (even though it is now automatically created by VS Code itself)